cog.md

```diff
@@ -52,57 +52,57 @@ The `symbols` section defines the script variables. Each variable is defined on
+| sector   | index in sector list    | Level sector                                                                                                                                                                               |
+| sound    | WAV filename            | Sound file                                                                                                                                                                                 |
+| sprite   | SPR filename            | Sprite file                                                                                                                                                                                |
+| surface  | index in surface list   | Level surface                                                                                                                                                                              |
+| vector   | (x/y/z)                 | Vector type                                                                                                                                                                                |
+| template | template name           | Game object template                                                                                                                                                                       |
+| thing    | index in the thing list | Game object                                                                                                                                                                                |
 
 *Note, all index values (e.g. for COG, sector, surface, thing) are usually defined by the level where COG script is used.*
 
 #### COG Symbol Reference Type
 Internal value of each symbol type is defined as fixed number. This number is retrieved in the script code for example by calling `GetSenderType` or `GetSourceType`. The following table defines the type values:
-| Type   | Value |
-|----------|-------------:|
-| COG_SYM_REF_NONE | 0 |
-| COG_SYM_REF_INT | 1 |
-| COG_SYM_REF_FLEX | 2 |
-| COG_SYM_REF_THING | 3 |
-| COG_SYM_REF_TEMPLATE | 4 |
-| COG_SYM_REF_SECTOR  | 5 |
-| COG_SYM_REF_SURFACE | 6
-| COG_SYM_REF_KEYFRAME| 7 |
-| COG_SYM_REF_SOUND  | 8 |
-| COG_SYM_REF_COG | 9 |
-| COG_SYM_REF_MATERIAL | 10 |
-| COG_SYM_REF_VECTOR | 11 |
-| COG_SYM_REF_MODEL | 12 |
-| COG_SYM_REF_AICLASS | 13 |
+| Type                 | Value |
+|----------------------|------:|
+| COG_SYM_REF_NONE     | 0     |
+| COG_SYM_REF_INT      | 1     |
+| COG_SYM_REF_FLEX     | 2     |
+| COG_SYM_REF_THING    | 3     |
+| COG_SYM_REF_TEMPLATE | 4     |
+| COG_SYM_REF_SECTOR   | 5     |
+| COG_SYM_REF_SURFACE  | 6     |
+| COG_SYM_REF_KEYFRAME | 7     |
+| COG_SYM_REF_SOUND    | 8     |
+| COG_SYM_REF_COG      | 9     |
+| COG_SYM_REF_MATERIAL | 10    |
+| COG_SYM_REF_VECTOR   | 11    |
+| COG_SYM_REF_MODEL    | 12    |
+| COG_SYM_REF_AICLASS  | 13    |
```

Comment:
The `COG_SYM_REF_` prefix should probably be stripped off from symbol types since that's engine's internal representation 🤔
Not that it's relevant for this pull request, just thinking out loud....

Reply:
Thanks for this PR and cleaning up the text and fix typos.